tools/NodeConfigCompare/configuration/cloud_formation_template.yaml

```diff
@@ -677,6 +679,20 @@ Resources:
       Description: boto3 latest
       LayerName: boto3-latest
       LicenseInfo: MIT
+  
+  StartUpLambdaLayer:
+    Type: AWS::Lambda::LayerVersion
+    DependsOn: CopyRedshiftConfigTestingCode
+    Properties:
+      CompatibleRuntimes:
+        - python3.7
+        - python3.8
+      Content:
+        S3Bucket: !Ref RedshiftConfigTestingBucket
+        S3Key: configurations/python.zip
```

Comment:
This S3 key seems very generic, can you make it specific to StartUpLambda?

Reply:
For this one I tried two file structures

```
cfnResponse_layer.zip
└── python
    ├── cfnresponse
    └── cfnresponse-1.1.2.dist-info
```
\
```
python.zip
├── cfnresponse
└── cfnresponse-1.1.2.dist-info
```

The second one worked. I have reached out to the Lambda-interest channel in slack for help.
I agree ideally having a structure similar to the first one is more understandable.

Reply:
Hmmm, i wonder why/how the naming affects the actual functionality. Let's wait on the response from the Lambda team.